force-app/main/default/classes/AssetManagementControllerTest.cls

```diff
@@ -0,0 +1,63 @@
+@IsTest(SeeAllData=true)
+public with sharing class AssetManagementControllerTest {
+    
+    private static void makeData(){
+        
+        //Inserting Product
+        Product2 product = new Product2(Name = 'Revenue Cloud Test Product 1', IsActive = true);
+        insert product;
+        
+        // Get the Standard Price Book ID
+        Id spb = Test.getStandardPricebookId();
+        //Updating Standard Price Book
+        update new Pricebook2(Id = spb , isActive = true);
+        
+ 
+        PricebookEntry priceBookEntry = new PricebookEntry(
+            Pricebook2Id = spb, Product2Id = product.Id, 
+            UnitPrice = 100, UseStandardPrice = false, IsActive = true);
+       
+        
+        insert priceBookEntry;
+        
+        
+        //Test Account
+        Account acc = new Account(Name = 'Revenue Cloud Test Account');
+        insert acc;
+        //Test Asset
+        Asset asset = new Asset(Name = 'Testing', AccountId = acc.Id, Product2Id = product.Id);
+        insert asset;
+    }
+    
+    @isTest static void testRenewAsset() {
+        makeData();
+        
+        List<Asset> assetList = [Select Id from Asset limit 1];
+        List<String> responseList;
+        if(!assetList.isEmpty()){
```

Comment:
So if you fail to insert data your test just... passes silently?

Reply:
I will add assert for the inserted data

Reply:
Now with your assertion, this `if` will always pass, so why have it?